authenticator/authenticator.go

```diff
@@ -0,0 +1,38 @@
+package authenticator
+
+import (
+	"github.com/gin-contrib/httpsign"
+	"github.com/gin-contrib/httpsign/crypto"
+	"github.com/gin-contrib/httpsign/validator"
+)
+
+// NewAuthenticator create a httpsign.Authenticator to check the message signing is valid or not
+func NewAuthenticator(readKeyID, readKeySecret, writeKeyID, writeKeySecret string) (*httpsign.Authenticator, error) {
```

Comment:
There are no error in this function, why do we return error here?

Reply:
fixed in 22dc64dfa78f517fcf47e19d433334d4e6e410f4.